src/angularjs/src/app/home/neighborhood-map.directive.js

```diff
@@ -19,7 +19,12 @@
         ctl.onMapReady = function (map) {
             ctl.map = map;
             Neighborhood.geojson().$promise.then(function (data) {
-                ctl.count = data && data.features ? data.features.length : '0';
+                if (data && data.features) {
```

Comment:
This tweak ensures that the JS on the site doesn't throw an exception when an empty feature collection with `features=null` is returned by the neighborhoods_geojson endpoint.